### PR TITLE
copy the past code

### PR DESCRIPTION
**Pull request by MaganaAsiati**
- [ ] Copy of my past project tribute page

### CODE REVIEW

**Comment by MaganaAsiati** [comment]
- [ ] kindly install linters in your project for Catching syntax errors is more efficient. 
to make the entire codebase looks like it was written by one person.
- [ ] There is no need to debug simple mistakes like typos - the linter does it for you.
Programmers can focus on solving problems instead of cleaning up the code.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️♻️♻️

Hi @MaganaAsiati,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

- [ ] You're doing great so far. It would be nice you make use of the GitHub inline comment feature and perform a review on the project as if it was done by someone else.
- [ ] Please use at least two(2) inline comments in the codebase that you can make comments on by telling the student to make changes to their code.
- [ ] Use this link [How to perform a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) given in the project requirement to guide you on how to go about it.
- [ ] Also, check this video out [How to perform code review with GitHub inline comment feature](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trubite Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="main">
```
Comment (MaganaAsiati) [inline index.html:10]: - [ ] you have done a great job here but it is a good practice 
to use proper elements such as header for this section etc.,
- [ ] Please develop the section properly using the right element:ok_hand:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants. </li>
+                <li>1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science. </li>
+                <li>1942 - Receives a Ph.D. in Genetics and Plant Pathology </li>
+                <li>1944 - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of disease resistent wheat - including different varieties for each major climate on Earth. </li>
+            </ul>
+        </section>
+           
+                
+    
+        <a href="" target="_blank" id="tribute-link">Trubite-Link</a>
```
Comment (MaganaAsiati) [inline index.html:33]: - [ ] kindly try to format your code by removing the unnecessary blank lines.
- [ ] It is always good practice to keep your code clean and good to read :+1:

File: style.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+#tribute-info ul{
+   text-align: left;
+}
+  a{ 
+    padding-bottom: 10px;
+    font-size: 20px;
+    text-decoration: none;
+    color: rgba(81, 81, 240, 0.753);
+}
```
Comment (MaganaAsiati) [inline style.css:56]: - [ ] You have done a great job here but it is a good practice 
to add the styles in related CSS files.
- [ ] please make  the necessary changes:ok_hand:

**Review by MaganaAsiati — COMMENTED** [review]
# changes required :
Please check my comments below and  make the required changes 
submit for the review when you are done 
Thank :+1:

File: index.html (hunk 1 above)
Comment (phelian23) [reply index.html:10]: Nicely done

File: index.html (hunk 2 above)
Comment (phelian23) [reply index.html:33]: Nicely done 👍

**Review by phelian23 — APPROVED** [review]
Hi @MaganaAsiati, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**